db_schema/migrations/12_anchor_impressions.sql

```diff
@@ -0,0 +1,38 @@
+-- Create table for storing aggregate impression metrics
+CREATE TABLE IF NOT EXISTS anchorImpressions (
+    account_id INTEGER NOT NULL,
+    date_start TIMESTAMP NOT NULL,
```

Comment:
we could use date instead of timestamp if we do not need the precision?

Reply:
oh, true. Changed.